front_end/entrypoints/rn_fusebox/rn_fusebox.ts

```diff
@@ -247,8 +247,8 @@ class FuseboxReactNativeApplicationObserver implements
     const {appDisplayName, deviceName} = event.data;
 
     // Update window title
-    if (appDisplayName != null) {
-      document.title = `${appDisplayName}${deviceName != null ? ` (${deviceName})` : ''} - React Native DevTools`;
+    if (appDisplayName !== null && appDisplayName !== undefined) {
```

Comment:
Bit unfortunate about this check — `!= null` is FB code style and more concise. Happy to align but we could consider overriding for `META_CODE_PATHS`.

Reply:
I am leaning more towards adopting what is already used in the project. Like what if, for example, we need to change non-FB codepaths? For hiding unsupported features, should we use FB code style or not?

I will keep it as it is now, but open to discuss the whole approach in a broader group, since we already have a list of things: `#` vs `private`, `!=` vs `!==`, ...

Reply:
I believe the existing repo convention is to use `if (object)` or `if(Boolean(…))`

Reply:
I've changed it this way, so there are no functional changes

e. g. `a != null` is the same as `a !== null && a !== undefined`, but not the same as `!!a` or `Boolean(a)`

Reply:
Yes the meaning would not the same for this specific line. I'm partial to adopting to local patterns too, so the `if (object)` or `if (Boolean(…))` seems to be the general convention here.

Tho in this specific case, it'd would seem like `if (appDisplayName)` would work slightly better as it'd filter out (already-trimmed) empty strings as well.

I'm totally happy that this PR as-is is scoped strictly on logical parity (same functional outcomes before and after lint fixes). Items mentioned in this thread is for future considerations when/if we do have a wider conversation around the style